src/learning-catalogue/model/factory/learningProviderFactory.ts

```diff
@@ -10,6 +10,8 @@ export class LearningProviderFactory {
 
 		learningProvider.id = data.id
 		learningProvider.name = data.name
+		learningProvider.cancellationPolicies = data.cancellationPolicies
+		learningProvider.termsAndConditions = data.termsAndConditions
```

Comment:
Do these need to be converted to the relevant objects?

Reply:
Nope, setting directly